### PR TITLE
first version for py-scrypt

### PR DESCRIPTION
**Pull request by holgern**
py-scrypt 0.8.3
https://bitbucket.org/mhallin/py-scrypt/

### CODE REVIEW

File: recipes/scrypt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - pkgconfig                    # [unix]
+    # For sha256
```
Comment (CJ-Wright) [inline recipes/scrypt/meta.yaml]: I'm confused.

File: recipes/scrypt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - pkgconfig                    # [unix]
```
Comment (CJ-Wright) [inline recipes/scrypt/meta.yaml]: Is this a build requirement?

File: recipes/scrypt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - scrypt
+  commands:
+    - conda inspect linkages -p $PREFIX scrypt  # [not win]
+    - conda inspect objects -p $PREFIX scrypt   # [osx]
+
+about:
+  home: https://bitbucket.org/mhallin/py-scrypt/
+  license: BSD-2-Clause
+  license_file: LICENSE
```
Comment (CJ-Wright) [inline recipes/scrypt/meta.yaml]: Needs a license_family

File: recipes/scrypt/meta.yaml (hunk 2 above)
Comment (holgern) [reply recipes/scrypt/meta.yaml]: I will try without pkconfig

File: recipes/scrypt/meta.yaml (hunk 1 above)
Comment (holgern) [reply recipes/scrypt/meta.yaml]: I will removed  # For sha256

File: recipes/scrypt/meta.yaml (hunk 3 above)
Comment (holgern) [reply recipes/scrypt/meta.yaml]: I will add this

**Comment by holgern** [comment]
Steem 0.18.103 #5100 needs scrypt with version 0.8.0.
When this recipe is accepted and build, I will update. 
Then, both version will be available, 0.8.0 and the newest one 0.8.3.

**Comment by CJ-Wright** [comment]
Seems resonable

**Pull request merged**